src/kudu/consensus/log_cache.cc

```diff
@@ -84,6 +84,14 @@ METRIC_DEFINE_gauge_int64(server, log_cache_msg_size, "Log Cache Message Size",
                           MetricUnit::kBytes,
                           "Size of the incoming uncompressed payload for the "
                           "messages in the log_cache");
+METRIC_DEFINE_gauge_int64(server, log_cache_compressed_payload_size,
+                         "Log Cache Compressed Payload Size",
+                          MetricUnit::kBytes,
+                          "Size of the compressed msg payload that is sent over the wire");
```

Comment:
If these will never be reset (pretty sure they will not be reset by Pushap's changes, right?) then we should make the description something more like "Total bytes of compressed message payloads sent over the wire since startup"

Reply:
I think they will be reset